server/src/lib/model/db/migrations/20191015151914-add-teams-and-user-client-teams.ts

```diff
@@ -0,0 +1,28 @@
+export const up = async function(db: any): Promise<any> {
+  return db.runSql(
+    `
+    CREATE TABLE teams (
+      team_id CHAR(36) NOT NULL PRIMARY KEY,
+      teamname TEXT NOT NULL DEFAULT (''),
+      leader CHAR(36) NOT NULL,
```

Comment:
I don't think teams will have leaders in the pilot – for now we can leave this out.